typescript/tsconfig.node18.json

```
@@ -4,9 +4,9 @@
   "display": "Vercel Node 18",
   "docs": "https://github.com/vercel/style-guide",
   "compilerOptions": {
-    "lib": ["es2023"],
+    "lib": ["esnext"],
     "module": "node16",
-    "target": "es2023",
+    "target": "esnext",
```

```suggestion
    "target": "es2022",
```
this should be 2022 based off of what I'm seeing here: https://github.com/tsconfig/bases/blob/main/bases/node18.json